doc/usersguide/cubit_trelis_plugin.rst

```diff
@@ -0,0 +1,37 @@
+Installing the Cubit/Trelis Plugin
+++++++++++++++++++++++++++++++++++
+
+Linux Install Instructions
+--------------------------
+Obtain the `DAGMC Plug-in <http://go.wisc.edu/dagmc-trelis>`_
+
+   1.  unpack this tar file in the plugin directory of Trelis, e.g. /opt/Trelis-16.0/bin/plugins/
+   2.  make a symbolic link in the plugin directory to the dagmc plugin itself. e.g. 
+       ln -s dagmc/libdagmc_export_plugin.so .
+
+OS/X Install Instructions
+-------------------------
+Obtain the `DAGMC Plug-in <https://drive.google.com/open?id=0B_tNonFwhOqQQkdhYWxWN1drVGM>`_
```

Comment:
I don't think this link is public, so perhaps it should be moved to a spot like the linux plugin at go.wisc.edu